yw_website/yw_website/apps/website/views.py

```diff
@@ -107,45 +110,30 @@ def yw_save_ping(request):
     return Response(status=200, data={"data": "You connected to yw web components."})
 
 
+@csrf_exempt
```

Comment:
is there a reason we need this for the create view, but not the update view?

Reply:
I added this because when I was creating these changes I kept running into very strange behaviors. I think we will need it for both as Django doesn't like POST requests without a CSRF token, and that would I imagine increase the complexity of the CLI tool

Reply:
I periodically run into the same issue and sometimes it effects me, sometimes it doesn't and I don't always see a reason why.  I tried to design the client on the CLI with this in mind.

Reply:
So leave it in?